packages/cli/src/lib/server.ts

```diff
@@ -203,6 +203,23 @@ function setupRoutes(def: DestinationDefinition | null): void {
     })
   )
 
+  router.post(
+    '/refreshAccessToken',
+    asyncHandler(async (req: express.Request, res: express.Response) => {
+      try {
+        const settings = {}
+        const data = await destination.refreshAccessToken(settings, req.body)
+        res.status(200).json({ ok: true, data })
+      } catch (e) {
+        const error = e as ResponseError
```

Comment:
Can we do something like this? This will return the entire response back if it is an HTTPError.
```suggestion
        const error = e as ResponseError
        const message = (await error?.response?.json()) ?? error.message
```